setup/setup.sh

```diff
@@ -7,6 +7,12 @@ then
     PRESERVECONFIG=1
 fi
 
+ISACTIVE=`systemctl is-active traccar.service`
```

Comment:
Can we remove `.service` suffix?

Reply:
This can be removed. Should I remove it also for the 'enable' line?

Reply:
Yep.